docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -164,4 +164,111 @@ Invoices must be paid **within 30 days of invoice receipt**, or no later than on
 Run of Show
 ===========
 
-We are still working out the run of show for the virtual event.
+This Run of Show provides more context about the event and answers some common questions you may have. 
+Please let us know if there is any information missing that would be useful for you.
+
+Sponsorship schedule
+--------------------
+
+* **SUNDAY**: The conference online platform is open. You are welcome to hang look at your sponsorship booth and attend the Writing Day, but no formal sponsorship events are happening. You're also welcome to lead a Writing Day project if your documentation is open source.
+
+* **MONDAY**: The conference platform opens at 8am, so we recommend arriving around this time to get the most interaction with attendees. This is the official start of the main conference talks, when sponsors tables will need to be staffed. The conference will run until around 5pm.
+
+* **TUESDAY**: The Job Fair will be Tuesday morning in the Sessions area of the online platform. The normal conference sponsorship activities and expo booths will be available all day Tuesday, but primarily on Tuesday afternoon. 
+
+See the :doc:`full schedule </conf/{{ shortcode }}/{{ year }}/schedule>` for exact timing details.
+
+Sponsorship platform
+--------------------
+
+We will be using `Hopin <https://hopin.to/>`_ as our online conference platform. It allows a number of unique spaces for attendees during the conference, and we hope it will allow a good amount of interaction between attendees and sponsors.
+
+**We will open the online platform 1 week prior to the conference starting**. At that point, you will be able to login and look around the platform, but it won't become fully active until the Sunday of the conference.
```

Comment:
This isn't accurate, because the only thing they'll be able to do is see the date/time of the event and set up their user accounts. Maybe we can reword it to reflect that?